src/components/XButton.js

```diff
@@ -0,0 +1,12 @@
+import styles from "./XButton.module.css";
+import { ReactComponent as XIcon } from "assets/images/ic_x.svg";
+
+const XButton = ({ onClick, className }) => {
```

Comment:
P2:
className 속성이 없을 경우 아래처럼 하시면 undefined가 문자열로 변환되어 들어갑니다. default값을 주시는 것을 추천드립니다.

```suggestion
const XButton = ({ onClick, className = '' }) => {
```